cocoa/BugsnagReactNative.m

```diff
@@ -215,7 +215,11 @@ + (void)startWithConfiguration:(BugsnagConfiguration *)config {
     NSString *appVersion = [RCTConvert NSString:options[@"appVersion"]];
     NSString *codeBundleId = [RCTConvert NSString:options[@"codeBundleId"]];
     BugsnagConfiguration* config = [Bugsnag bugsnagStarted] ? [Bugsnag configuration] : [BugsnagConfiguration new];
-    config.apiKey = apiKey;
+
+    if (apiKey != nil && apiKey.length > 0) {
```

Comment:
do we need to check for `nil` here? It seems like on line 208 we just check the length

Reply:
No, if the value is `nil`, then length is coerced to 0.